src/shapes/plane.js

```diff
@@ -1,22 +1,22 @@
 import { Entity } from '../core/entity.js';
 
 const vertices = [
-  -1,  1,  1,
-   1,  1,  1,
-   1, -1,  1,
-  -1, -1,  1
+  -1,  0,  1,
+   1,  0,  1,
+   1, 0,  -1,
+  -1, 0,  -1
```

Comment:
In the new XYZ coordinates, the plane was rendered vertically rather than horizontally.